ME3Explorer/NativesEditor/Views/StateEventMapView.xaml.cs

```diff
@@ -508,7 +520,29 @@ private void AddSubstateSiblingIndex_Click(object sender, System.Windows.RoutedE
 
         private void RemoveSubstateSiblingIndex_Click(object sender, System.Windows.RoutedEventArgs e)
         {
-            RemoveSubstateSiblingIndex(SubstateStateEventSiblingIndicesListBox.SelectedIndex);
+            var button = (Button)sender;
+            RemoveSubstateSiblingIndex((int)button.Tag);
+        }
+    }
+
+    /// <summary>
+	///   Resolves a name entry for a given Pcc
+	/// </summary>
+    public class NameConverter : IMultiValueConverter
+    {
+        public object Convert(object[] values, Type targetType, object parameter, CultureInfo culture)
+        {
+            if (values[0] is IMEPackage pcc && values[1] is int index)
+            {
+                return new NameReference(pcc.GetNameEntry(index), (values[2] as int) ?? 0).Instanced;

```

Comment:
Aaand that suggestion broke the build :weary: . Guess that's what I get for trying to write code in github instead of an IDE. Surely it will work this time!

```suggestion
            if (values[0] is IMEPackage pcc && values[1] is int index && values[2] is int instanceNum)
            {
                return new ME3ExplorerCore.Unreal.NameReference(pcc.GetNameEntry(index), instanceNum).Instanced;

```

Sorry for cluttering up your pull request like this.